### ISSUE TITLE
NoSuchMethodException on every join

### ISSUE THREAD

**Issue by Sytm** [wagyourtail/WagYourMinimap#6]
```
[14:22:26] [Render thread/ERROR]: Error executing task on Client
java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.NoSuchMethodException: net.minecraft.class_2188.getSummonableEntity(com.mojang.brigadier.context.CommandContext,java.lang.String)
	at xyz.wagyourtail.config.command.SettingCommand.getCommandTree(SettingCommand.java:56) ~[wagyourminimap-1.19-1.0.8-fabric.jar:?]
	at xyz.wagyourtail.config.command.SettingCommand.getCommandTree(SettingCommand.java:35) ~[wagyourminimap-1.19-1.0.8-fabric.jar:?]
	at xyz.wagyourtail.minimap.client.WagYourMinimapClient.lambda$init$6(WagYourMinimapClient.java:211) ~[wagyourminimap-1.19-1.0.8-fabric.jar:?]
	at xyz.wagyourtail.minimap.fabric.WagYourMinimapFabric.lambda$onInitializeClient$2(WagYourMinimapFabric.java:21) ~[wagyourminimap-1.19-1.0.8-fabric.jar:?]
	at net.fabricmc.fabric.api.client.command.v2.ClientCommandRegistrationCallback.lambda$static$0(ClientCommandRegistrationCallback.java:36) ~[fabric-command-api-v2-2.1.2+0d55f585a9-c900b1bf2afa8f42.jar:?]
	at net.minecraft.class_634.handler$zgb000$onGameJoin(class_634.java:3959) ~[client-intermediary.jar:?]
	at net.minecraft.class_634.method_11120(class_634.java:397) ~[client-intermediary.jar:?]
	at net.minecraft.class_2678.method_11567(class_2678.java:86) ~[client-intermediary.jar:?]
	at net.minecraft.class_2678.method_11054(class_2678.java:20) ~[client-intermediary.jar:?]
	at net.minecraft.class_2600.method_11072(class_2600.java:22) ~[client-intermediary.jar:?]
	at net.minecraft.class_1255.method_18859(class_1255.java:157) [client-intermediary.jar:?]
	at net.minecraft.class_4093.method_18859(class_4093.java:23) [client-intermediary.jar:?]
	at net.minecraft.class_1255.method_16075(class_1255.java:131) [client-intermediary.jar:?]
	at net.minecraft.class_1255.method_5383(class_1255.java:116) [client-intermediary.jar:?]
	at net.minecraft.class_310.method_1523(class_310.java:1095) [client-intermediary.jar:?]
	at net.minecraft.class_310.method_1514(class_310.java:734) [client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:237) [client-intermediary.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461) [fabric-loader-0.14.8.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) [fabric-loader-0.14.8.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.14.8.jar:?]
	at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:577) ~[?:?]
	at org.polymc.impl.OneSixLauncher.invokeMain(OneSixLauncher.java:104) [NewLaunch.jar:?]
	at org.polymc.impl.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:175) [NewLaunch.jar:?]
	at org.polymc.impl.OneSixLauncher.launch(OneSixLauncher.java:185) [NewLaunch.jar:?]
	at org.polymc.EntryPoint.listen(EntryPoint.java:144) [NewLaunch.jar:?]
	at org.polymc.EntryPoint.main(EntryPoint.java:74) [NewLaunch.jar:?]
Caused by: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.NoSuchMethodException: net.minecraft.class_2188.getSummonableEntity(com.mojang.brigadier.context.CommandContext,java.lang.String)
	at xyz.wagyourtail.config.command.SettingCommand.getSubCommandBuilder(SettingCommand.java:325) ~[wagyourminimap-1.19-1.0.8-fabric.jar:?]
	at xyz.wagyourtail.config.command.SettingCommand.getCommandTreeForSingle(SettingCommand.java:268) ~[wagyourminimap-1.19-1.0.8-fabric.jar:?]
	at xyz.wagyourtail.config.command.SettingCommand.getCommandTree(SettingCommand.java:76) ~[wagyourminimap-1.19-1.0.8-fabric.jar:?]
	at xyz.wagyourtail.config.command.SettingCommand.getCommandTree(SettingCommand.java:47) ~[wagyourminimap-1.19-1.0.8-fabric.jar:?]
	... 26 more
Caused by: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.NoSuchMethodException: net.minecraft.class_2188.getSummonableEntity(com.mojang.brigadier.context.CommandContext,java.lang.String)
	at xyz.wagyourtail.config.command.SettingCommand.getSubCommandBuilder(SettingCommand.java:325) ~[wagyourminimap-1.19-1.0.8-fabric.jar:?]
	at xyz.wagyourtail.config.command.SettingCommand.getCommandTreeForArray(SettingCommand.java:158) ~[wagyourminimap-1.19-1.0.8-fabric.jar:?]
	at xyz.wagyourtail.config.command.SettingCommand.getCommandTree(SettingCommand.java:70) ~[wagyourminimap-1.19-1.0.8-fabric.jar:?]
	at xyz.wagyourtail.config.command.SettingCommand.getSubCommandBuilder(SettingCommand.java:309) ~[wagyourminimap-1.19-1.0.8-fabric.jar:?]
	at xyz.wagyourtail.config.command.SettingCommand.getCommandTreeForSingle(SettingCommand.java:268) ~[wagyourminimap-1.19-1.0.8-fabric.jar:?]
	at xyz.wagyourtail.config.command.SettingCommand.getCommandTree(SettingCommand.java:76) ~[wagyourminimap-1.19-1.0.8-fabric.jar:?]
	at xyz.wagyourtail.config.command.SettingCommand.getCommandTree(SettingCommand.java:47) ~[wagyourminimap-1.19-1.0.8-fabric.jar:?]
	... 26 more
Caused by: java.lang.RuntimeException: java.lang.NoSuchMethodException: net.minecraft.class_2188.getSummonableEntity(com.mojang.brigadier.context.CommandContext,java.lang.String)
	at xyz.wagyourtail.config.command.SettingCommand.getSubCommandBuilder(SettingCommand.java:325) ~[wagyourminimap-1.19-1.0.8-fabric.jar:?]
	at xyz.wagyourtail.config.command.SettingCommand.getCommandTreeForSingle(SettingCommand.java:268) ~[wagyourminimap-1.19-1.0.8-fabric.jar:?]
	at xyz.wagyourtail.config.command.SettingCommand.getCommandTree(SettingCommand.java:76) ~[wagyourminimap-1.19-1.0.8-fabric.jar:?]
	at xyz.wagyourtail.config.command.SettingCommand.getSubCommandBuilder(SettingCommand.java:309) ~[wagyourminimap-1.19-1.0.8-fabric.jar:?]
	at xyz.wagyourtail.config.command.SettingCommand.getCommandTreeForArray(SettingCommand.java:158) ~[wagyourminimap-1.19-1.0.8-fabric.jar:?]
	at xyz.wagyourtail.config.command.SettingCommand.getCommandTree(SettingCommand.java:70) ~[wagyourminimap-1.19-1.0.8-fabric.jar:?]
	at xyz.wagyourtail.config.command.SettingCommand.getSubCommandBuilder(SettingCommand.java:309) ~[wagyourminimap-1.19-1.0.8-fabric.jar:?]
	at xyz.wagyourtail.config.command.SettingCommand.getCommandTreeForSingle(SettingCommand.java:268) ~[wagyourminimap-1.19-1.0.8-fabric.jar:?]
	at xyz.wagyourtail.config.command.SettingCommand.getCommandTree(SettingCommand.java:76) ~[wagyourminimap-1.19-1.0.8-fabric.jar:?]
	at xyz.wagyourtail.config.command.SettingCommand.getCommandTree(SettingCommand.java:47) ~[wagyourminimap-1.19-1.0.8-fabric.jar:?]
	... 26 more
Caused by: java.lang.NoSuchMethodException: net.minecraft.class_2188.getSummonableEntity(com.mojang.brigadier.context.CommandContext,java.lang.String)
	at java.lang.Class.getMethod(Class.java:2259) ~[?:?]
	at xyz.wagyourtail.config.command.SettingCommand.getCommandTreeForArray(SettingCommand.java:91) ~[wagyourminimap-1.19-1.0.8-fabric.jar:?]
	at xyz.wagyourtail.config.command.SettingCommand.getCommandTree(SettingCommand.java:70) ~[wagyourminimap-1.19-1.0.8-fabric.jar:?]
	at xyz.wagyourtail.config.command.SettingCommand.getSubCommandBuilder(SettingCommand.java:309) ~[wagyourminimap-1.19-1.0.8-fabric.jar:?]
	at xyz.wagyourtail.config.command.SettingCommand.getCommandTreeForSingle(SettingCommand.java:268) ~[wagyourminimap-1.19-1.0.8-fabric.jar:?]
	at xyz.wagyourtail.config.command.SettingCommand.getCommandTree(SettingCommand.java:76) ~[wagyourminimap-1.19-1.0.8-fabric.jar:?]
	at xyz.wagyourtail.config.command.SettingCommand.getSubCommandBuilder(SettingCommand.java:309) ~[wagyourminimap-1.19-1.0.8-fabric.jar:?]
	at xyz.wagyourtail.config.command.SettingCommand.getCommandTreeForArray(SettingCommand.java:158) ~[wagyourminimap-1.19-1.0.8-fabric.jar:?]
	at xyz.wagyourtail.config.command.SettingCommand.getCommandTree(SettingCommand.java:70) ~[wagyourminimap-1.19-1.0.8-fabric.jar:?]
	at xyz.wagyourtail.config.command.SettingCommand.getSubCommandBuilder(SettingCommand.java:309) ~[wagyourminimap-1.19-1.0.8-fabric.jar:?]
	at xyz.wagyourtail.config.command.SettingCommand.getCommandTreeForSingle(SettingCommand.java:268) ~[wagyourminimap-1.19-1.0.8-fabric.jar:?]
	at xyz.wagyourtail.config.command.SettingCommand.getCommandTree(SettingCommand.java:76) ~[wagyourminimap-1.19-1.0.8-fabric.jar:?]
	at xyz.wagyourtail.config.command.SettingCommand.getCommandTree(SettingCommand.java:47) ~[wagyourminimap-1.19-1.0.8-fabric.jar:?]
	... 26 more
```

I get this error message every time I join a server or local world, but only once.

This issue also appears when using Quilt with Quilted Fabric API, but instead of silent errors on join it happens at startup and crashes the process. [crash-2022-07-24_15.17.50-client.txt](https://github.com/wagyourtail/WagYourMinimap/files/9175858/crash-2022-07-24_15.17.50-client.txt)

Running Minecraft 1.19, Fabric Loader 0.14.8, Fabric API 0.58.0, ArchitecturyAPI 5.10.33 launched with PolyMC on Linux

**Comment by wagyourtail** [comment]
I think I fixed this with the 1.0.9 release, can you test?

**Comment by Sytm** [comment]
Works flawless, even on Quilt!
